libs/blocks/table/table.js

```diff
@@ -368,6 +368,14 @@ function handleSection(sectionParams) {
       sectionRowTitle.setAttribute('scope', 'row');
     }
   }
+
+  rowCols.forEach((col) => {
+    const textModifiers = ['STRONG', 'EM', 'MARK', 'B', 'I', 'U', 'DEL', 'INS', 'CODE', 'SMALL', 'SUB', 'SUP', 'CITE'];
```

Comment:
I'm only aware of these styles that are applied through the EDS pipeline (excluding headings, links and all the obvious ones) - https://main--milo--adobecom.aem.page/drafts/ramuntea/word-styles. So `strong`, `em`, `u`, `code`, `del`, `sub`, `sup`. Were you able to get a `mark`, `b`, `i`, `u`, `ins`, `small` or `cite`? If not, we can probably remove those references.